.circleci/config.yml

```diff
@@ -232,7 +232,9 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
-
+            - run:
+                name: Close the Release Milestone
+                command: scripts/close-milestone.sh v$(extract_version_from_current_wd)
```

Comment:
Hmm is `extract_version_from_current_wd` available in this context? Pls test (for the tests, move this step out of the `when` block)

Reply:
Thanks for bringing this up. I was thinking of assigning the value of `extract_version_from_current_wd` before the release to a value in the release script, and move the logic to close the associated milestone to the end of the release script. What do you think?

Reply:
Okay, let's give it a shot 👌🏻